lib/chiketto/user.rb

```diff
@@ -22,6 +22,11 @@ def events(params = {})
       events.map { |e| Event.new e }
     end
 
+    def event_attendees(params = {})
+      attendees = User.paginated_attendees @id, params
```

Comment:
This method (`paginated_attendess`) doesn't exist — I'm also not sure why a `User` would have `attendees`, since they should belong to an event, could you clarify what you're trying to do here?